combat-trainer.lic

```diff
@@ -266,9 +266,7 @@ class SetupProcess
     elsif !next_summoned && !DRStats.moon_mage?
       break_summoned_weapon(game_state.last_weapon_name)
     elsif !next_summoned && DRStats.moon_mage?
-      if right_hand =~ /moon/ || left_hand =~ /moon/
-        bput('wear moon', 'telekinetic')
-      end
+      wear_moon_weapon
```

Comment:
I'd prefer to see the module prefix (`DRCI.wear_moon_weapon`), but it's not a big deal

Reply:
Sure thing